src/common/theme/components/ButtonTheme.ts

```diff
@@ -30,6 +30,11 @@ export const ButtonTheme = defineStyleConfig({
         border: '2px solid',
         borderColor: 'blue.400',
       },
+      _disabled: {
```

Comment:
Could you move disabled to hover above?

```
_hover: {
        bgColor: 'gray.900',
        _disabled: {
          bg: 'gray.900',
        },
      },
```

This way we can avoid using `!important`